config/webpack.prod.js

```diff
@@ -29,6 +29,7 @@ const OfflinePlugin = require('offline-plugin');
  * Webpack Constants
  */
 const ENV = process.env.NODE_ENV = process.env.ENV = 'production';
+const FABRIC8_FORGE_API_URL = process.env.FABRIC8_FORGE_API_URL || 'https://forge.api.prod-preview.openshift.io';
```

Comment:
I think this should be configured in the docker container or somewhere else. 

@pmuir where did you put the other URLs?